palette/README.md

```diff
@@ -53,6 +53,7 @@ These features are disabled by default:
 * `"libm"` - Uses the [`libm`] floating point math library (for when the `std` feature is disabled).
 * `"bytemuck"` - Enables casting between plain data types using [`bytemuck`].
 * `"wide"` - Enables support for using SIMD types from [`wide`].
+* `"simba"` - Enables support for using (the wrapped [`wide`]) SIMD types from [`simba`].
```

Comment:
I suppose it's a bit odd to not support the stable none-wide types they have. 🤔 Would that be possible to include, considering it's already implemented with macros?